examples/sinh_libtorch/README.md

```diff
@@ -3,7 +3,7 @@
 ```
 mkdir build
 cd build
-cmake -DCMAKE_PREFIX_PATH="$(python -c 'import torch.utils; print(torch.utils.cmake_prefix_path)')" ..
+cmake -DCMAKE_PREFIX_PATH="$(python -c 'import nvfuser; import torch; print(nvfuser.cmake_prefix_path, torch.utils.cmake_prefix_path, sep=";")')" ..
```

Comment:
I'm not good with python. is `import torch; ... torch.utils ...` the same as `improt torch.utils`?

Reply:
mostly asking this since I thought torch.utils is a lightweight import... But I also think I'm totally talking non-sense there.